tests_integration_with_local_chain/markets/seer/test_seer_agent_market.py

```
@@ -1,22 +1,62 @@
 from prediction_market_agent_tooling.config import APIKeys
+from prediction_market_agent_tooling.gtypes import xdai_type
+from prediction_market_agent_tooling.markets.agent_market import (
+    FilterBy,
+    ProcessedTradedMarket,
+    SortBy,
+)
+from prediction_market_agent_tooling.markets.blockchain_utils import store_trades
+from prediction_market_agent_tooling.markets.data_models import (
+    PlacedTrade,
+    ProbabilisticAnswer,
+    TokenAmount,
```

Ruff (0.8.2)

`prediction_market_agent_tooling.markets.data_models.TokenAmount` imported but unused

Remove unused import: `prediction_market_agent_tooling.markets.data_models.TokenAmount`

(F401)